src/processor/operator/partitioner.cpp

```diff
@@ -31,6 +31,9 @@ void PartitionerSharedState::initialize() {
     KU_ASSERT(maxNodeOffsets.size() == 2);
     numPartitions.resize(maxNodeOffsets.size());
     for (auto i = 0u; i < maxNodeOffsets.size(); i++) {
+        if (!copyNodeSharedStates.empty()) {
```

Comment:
Can you leave a todo for me to also move the setting of maxNodeOffsets inside `PartitionerSharedState `? I think we can unify the case for rdf and property graph tables.